test/Quake/merge_rotation.qke

```diff
@@ -85,17 +83,16 @@ func.func @duplicate_rotation_check3() {
 }
 
 // CHECK-LABEL:   func.func @duplicate_rotation_check3() {
-// CHECK-DAG:       %[[VAL_0:.*]] = arith.constant 6.400000e-01 : f64
-// CHECK-DAG:       %[[VAL_1:.*]] = arith.constant 2.300000e-01 : f64
-// CHECK-DAG:       %[[VAL_2:.*]] = arith.constant 5.900000e-01 : f64
-// CHECK-DAG:       %[[VAL_3:.*]] = quake.alloca !quake.veq<2>
-// CHECK:           %[[VAL_4:.*]] = quake.extract_ref %[[VAL_3]][0] : (!quake.veq<2>) -> !quake.ref
-// CHECK:           %[[VAL_5:.*]] = call @returns_angle(%[[VAL_2]], %[[VAL_1]]) : (f64, f64) -> f64
-// CHECK:           %[[VAL_6:.*]] = call @returns_angle(%[[VAL_0]], %[[VAL_1]]) : (f64, f64) -> f64
-// CHECK:           quake.rx (%[[VAL_2]]) %[[VAL_4]] : (f64, !quake.ref) -> ()
-// CHECK:           quake.rx (%[[VAL_6]]) %[[VAL_4]] : (f64, !quake.ref) -> ()
-// CHECK:           quake.rx (%[[VAL_5]]) %[[VAL_4]] : (f64, !quake.ref) -> ()
-// CHECK:           %[[VAL_7:.*]] = quake.mz %[[VAL_4]] : (!quake.ref) -> i1
+// CHECK-DAG:       %[[VAL_0:.*]] = arith.constant 2.782{{.*}} : f64
+// CHECK-DAG:       %[[VAL_1:.*]] = arith.constant 2.565{{.*}} : f64
+// CHECK-DAG:       %[[VAL_2:.*]] = arith.constant 5.900{{.*}} : f64
```

Comment:
I think we should keep the `e-01` (assuming that's that it should be) as part of check. Otherwise we risk being off by an order of magnitude.